### ISSUE TITLE
MacOS: Euroscope fails to run on wine env.

### ISSUE THREAD

**Issue by EdV2008** [jonaseberle/euroscope-afv-wine#12]
After installing wine, I tried to download euroscope and it worked as I have the euroscope file, but when it came to actually installing it on my macbook, it seemed to fail and leading into a continuous loop of trying to install it. Does anyone know how it can be fixed.

Terminal transcript attached below if anyone can find a solution.

[message.txt](https://github.com/user-attachments/files/17269953/message.txt)

(downloadable)

**Comment by EdV2008** [comment]
@tgpetrica

**Comment by jonaseberle** [comment]
The log you sent is very convoluted and hard to follow. It contains several commands that are not relevant at all to your problem. You also did not follow the commands from the instructions at all.
Can you describe more concisely what you did and what happens?

Maybe other Mac users can confirm if it is currently working for them.

**Comment by EdV2008** [comment]
Myself not being the genius with macbooks, I tried to work with the developer of rovacc about installing and getting ES to work. I myself am not too certain what went wrong or what I did to a full extent as I followed the developers instructions to try and install. Currently both me and him don't know how to proceed further and will be trying to change by me moving to a windows laptop instead.

**Comment by jonaseberle** [comment]
Moving to Windows is the right thing to do I think. You are struggling and this is a lot easier with some background knowledge. I wish you all the best for your controlling! And big respect to RoVACC for supporting you in this endeavour.

**Comment by tgpetrica** [comment]
> Moving to Windows is the right thing to do I think. You are struggling and this is a lot easier with some background knowledge. I wish you all the best for your controlling! And big respect to RoVACC for supporting you in this endeavour.

Hello Jonas,

I was helping our trainee set up EuroScope on a Mac system, but unfortunately, we couldn't get it to work. It's interesting that we can natively run voice communication apps on non-Windows systems, yet we're still limited when it comes to a controller clients. Anyway, thanks a lot for the assistance you offered!

Regards,
Teodor Petrica
